options/db_options.cc

```diff
@@ -1011,7 +1019,8 @@ MutableDBOptions::MutableDBOptions(const DBOptions& options)
       wal_bytes_per_sync(options.wal_bytes_per_sync),
       strict_bytes_per_sync(options.strict_bytes_per_sync),
       compaction_readahead_size(options.compaction_readahead_size),
-      max_background_flushes(options.max_background_flushes) {}
+      max_background_flushes(options.max_background_flushes),
+      daily_offpeak_time_utc(options.daily_offpeak_time_utc) {}
```

Comment:
Since this time is really only HH:mm, should it just be "daily_offpeak_time" rather than "daily_offpeak_time_utc"?

Reply:
We want to explicitly say that it needs to be in UTC. Many of the internal configurations are often in local timezone (Pacific). Without `_utc` suffix, users may wrongfully assume that the value should be in the  local timezone, not UTC.

For example, let's say the server is running in PDT and the user wants to set the offpeak period from 11:30PM to 4:30AM next day. If they set `23:30-04:30`, `IsNowOffPeak()` will return true from 4:30PM to 9:30PM PDT which is not what the user intended. What they actually want to set is `06:30-11:30` which is in UTC.

Reply:
And yes, it's an unfortunate extra burden for users to figure out the conversion and set the correct value in UTC. Per internal discussion, we concluded that fixing the value in UTC is better than our options dependent on the configured time zone... (see also @pdillinger 's comment - https://github.com/facebook/rocksdb/pull/11893#pullrequestreview-1647346844)